msfvenom

```diff
@@ -177,6 +177,10 @@ def parse_args(args)
     opts[:add_code] = x
   end
 
+  opt.on('-z', '--prepend-code        <path>', String, 'Specify shellcode file to append') do |x|
```

Comment:
Why `-z`,  for the short flag? This does to have a clear logical connection to the longer form to me.

Reply:
Maybe just support a long flag for initial pass and if we find high usages occurs promote to add a short flag when/if it makes sense.